tools/IoTEdgeDevOps/VstsPipelineSync/Program.cs

```diff
@@ -0,0 +1,28 @@
+namespace VstsPipelineSync
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using DevOpsLib;
+
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            (string pat, string dbConnectionString, TimeSpan waitPeriodBeforeNextUpdate) = GetInputsFromArgs(args);
+            Console.WriteLine($"Wait period before next update=[{waitPeriodBeforeNextUpdate}]");
+            await new VstsBuildBatchUpdate(new DevOpsAccessSetting(pat), dbConnectionString).RunAsync(waitPeriodBeforeNextUpdate, CancellationToken.None);
+        }
+
+        private static (string pat, string dbConnectionString, TimeSpan waitPeriodBeforeNextUpdate) GetInputsFromArgs(string[] args)
+        {
+            if (args.Length != 3)
```

Comment:
What's the format of this `args` gonna look like? What's in it? Maybe add a comment there on what format of data is expecting?